lib/dom.js

```diff
@@ -751,6 +752,20 @@ declare class HTMLFormElement extends HTMLElement {
     submit(): void;
 }
 
+declare class HTMLIFrameElement extends HTMLElement {
+    allowFullScreen: boolean;
+    frameBorder: number;
```

Comment:
i think frameBorder, height, marginHeight, marginWidth, and width are all strings, per https://www.w3.org/TR/DOM-Level-2-HTML/html.html#ID-50708718 and also tried this in Chrome:

```
var frame = document.createElement('iframe');
frame.width // ""
frame.height // ""
// ...
```